uploadNeuroDB/minc_insertion.pl

```diff
@@ -387,7 +390,11 @@
 $file->setFileData('OutputType', 'native');
 $file->setFileData('FileType', 'mnc');
 $file->setFileData('TarchiveSource', $tarchiveInfo{'TarchiveID'});
-$file->setFileData('Caveat', 0);
+if (defined($seriesuid)) {
+    $file->setFileData('Caveat', 1);
```

Comment:
@gluneau 

the seriesUID is set when we are deleting a file (as in this pull request: https://github.com/aces/Loris-MRI/pull/138/files)
I was thinking we should set Caveat to 1 when we are insering a violated scan (referring to this pull request: https://github.com/aces/Loris-MRI/pull/141)